### PR TITLE
Switched to UDP and Globalized Messaging

### LINKED ISSUE #82: 2 New (Constant) Ports For Messages
1 For Blockchain Messages
1 For Relay Messages

### LINKED ISSUE #83: Delineate Session Message based on DevID

### LINKED ISSUE #85: Basic message handler structure (Global)

### PR DESCRIPTION
**Pull request by andrewnguyen22**
closes #82 
closes #83 
closes #85

### CODE REVIEW

File: message/handlers.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	"fmt"
+	"github.com/pokt-network/pocket-core/session"
+	"net"
+	"sync"
+)
+
+func HandleMessage(message *Message, addr *net.UDPAddr){
+	switch message.Payload.ID {
+	case 0: // simple print message (testing purposes)
+		PrintMessage(message, addr)
```
Comment (luyzdeleon) [inline message/handlers.go]: Can't we just print all incoming messages? This could be switched on and off by a config in the future.

**Review by luyzdeleon — CHANGES_REQUESTED** [review]

File: message/handlers.go (hunk 1 above)
Comment (andrewnguyen22) [reply message/handlers.go]: Yes, so ultimately this is just a testing thing! I can remove if it is too cluttered

Comment (luyzdeleon) [reply message/handlers.go]: I just feel it's more convenient to print all messages by default, so you don't have to send 2 messages to test.

Comment (andrewnguyen22) [reply message/handlers.go]: Updated and removed

**Review by luyzdeleon — APPROVED** [review]

**Pull request merged**